### ISSUE TITLE
Handle multi proposals selection

### ISSUE THREAD

**Issue by JustinCast** [EveripediaNetwork/iq-bridge-ui#72]

**Comment by kesar** [comment]
isnt this already done?

**Comment by JustinCast** [comment]
> isnt this already done?

No, this hasn't been done, cause we need to refactorize some of our existent code. Since every proposal can has several options, We need to think about a way to manage or get the user through the voting process. You Know what I mean?
